api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry;
+
+import io.opentelemetry.spi.TracerProvider;
+import io.opentelemetry.trace.NoopTrace;
+import io.opentelemetry.trace.Trace;
+import io.opentelemetry.trace.Tracer;
+import java.util.ServiceLoader;
+
+/**
+ * This class provides a static global accessor for telemetry objects {@link Tracer}, {@link
```

Comment:
I would like to have clear documented guidance when this is meant to be used and when not.

Reply:
+1, I think all docs will reference this class. I have done some changes in #252 which changes docs in `Tracer`